### PR TITLE
GitHub error statuses

### PR DESCRIPTION
**Pull request by camertron**
Indicate an error status on pull requests if an error occurs while pushing content to Transifex.

@lumoslabs/platform

### CODE REVIEW

File: server/lib/txgh-server/webhooks/github/status_updater.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+              update_status_safely do
+                Txgh::GithubStatus.error(project, repo, branch, status_params)
+              end
+            end
+          end
+        end
+
+        def update_status
+          update_status_safely do
+            Txgh::GithubStatus.update(project, repo, branch)
```
Comment (zvkemp) [inline server/lib/txgh-server/webhooks/github/status_updater.rb:29]: Where is this from and what does it do?

Comment (camertron) [reply server/lib/txgh-server/webhooks/github/status_updater.rb:29]: It's a class that lives in txgh, here's a [link](https://github.com/lumoslabs/txgh/blob/master/txgh/lib/txgh/github_status.rb). It's the thing that actually calls Github's status API.

Comment (camertron) [reply server/lib/txgh-server/webhooks/github/status_updater.rb:29]: I suppose `StatusUpdater` and `GithubStatus` could be merged into a single class. Is that what you're hinting at?

Comment (zvkemp) [reply server/lib/txgh-server/webhooks/github/status_updater.rb:29]: No, I just couldn't find it. For some reason the search didn't locate it.

**Review by zvkemp — APPROVED** [review]

**Pull request merged**